e2e/browser/testApp/package.json

```
@@ -16,30 +16,16 @@
     "start": "next start"
   },
   "dependencies": {
-    "@inrupt/prism-react-components": "^0.13.1-alpha.2",
     "@inrupt/solid-client": "^1.20.1",
     "@inrupt/solid-client-access-grants": "file:../../../",
     "@inrupt/solid-client-authn-browser": "^1.12.2",
```

```suggestion
    "@inrupt/solid-client-authn-browser": "^1.12.3",
```

Just to save the dependabot PR

---

61aaea9d6e70e95ce06380a9f6cc11660934e12b